src/chesster.js

```diff
@@ -547,6 +547,7 @@ function(bot,message) {
 /* mods */
 
 leagueResponse(['summon mods'], 'formatSummonModsResponse');
+leagueResponse(['notify mods'], 'formatSummonModsResponse');
```

Comment:
Meaning you can simplify that to: `leagueResponse(['summon  mods', 'notify mods', 'holla at mods'], 'formatSummonModsResponse');`